cabal.project

```diff
@@ -47,5 +47,13 @@ package cardano-crypto-praos
   flags: -external-libsodium-vrf
 
 constraints:
-  cardano-api ^>= 8.36
-  , quickcheck-contractmodel ^>= 0.1.6
+  cardano-api ^>= 8.42
+
+source-repository-package
```

Comment:
Why is an SRP needed for this?

Reply:
Both this package and quickcheck-contractmodel hardcode the current era. So to keep making it possible to test Babbage era while Mainnet is still on it, the Conway code lives in a branch: https://github.com/input-output-hk/quickcheck-contractmodel/tree/Conway

When Mainnet switches to Conway, the Conway branch of quickcheck-contractmodel can be merged, a new version can be published on CHaP and the SRP can be removed.